docs/config/aws_template.md

```diff
@@ -0,0 +1,431 @@
+
+=== "CodeBuild steps"
+
+    The following image shows how code changes are deployed to AWS and the steps that make up the AWS CodeBuild process
+    to update or create lambda functions and containers for each pipeline.
+
+    ![Image](../figures/aws/code_build.png)
+
+## Prerequisites
+
+### Create code repos
+
+Make sure that you have two repositories in your GitHub organization/account created from the following templates:
+
+* **<https://github.com/tsdat/pipeline-template>**
+* **<https://github.com/tsdat/aws-template>**
+
+If you are using an existing `pipeline-template` repository, make sure that the `requirements.txt` file specifies a
+`tsdat` version of at least `tsdat==0.7.1`. The AWS build will not work with earlier versions of `tsdat`.
+
+Clone these repos to the same parent folder on your computer.
+
+???+ warning "Warning: Windows users"
+
+    If you are using WSL on Windows make sure you run the `git clone` command from a WSL terminal.
+
+### Install Docker
+
+We use a Docker container with VSCode to make setting up your development environment a snap.  We assume users have a
+basic familiarity with Docker containers. If you are new to Docker, there are many free online tutorials to get you
+started.
+
+!!! note
+
+    Docker Desktop can be flaky, especially on Windows, and it requires a license so we recommend not using it. Instead,
+    we are providing alternative, non-Docker Desktop installation instructions for each platform. The Docker Desktop
+    install is easier and requires fewer steps, so it may be fine for your needs, but keep in mind it may crash if you
+    update it (requiring a full uninstall/reinstall, and then you lose all your container environments).
+
+    === "Windows Users"
+
+        * [Install Docker on wsl2](https://github.com/clansing/docs/blob/main/windows-docker-wsl2.md)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/windows-install/)
+
+    === "Mac Users"
+
+        * [Use Docker/Colima](https://dev.to/elliotalexander/how-to-use-docker-without-docker-desktop-on-macos-217m)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/mac-install/)
+
+    === "Linux Users"
+
+        * [Install Docker](https://docs.docker.com/engine/install/ubuntu/)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/linux-install/)
+
+We also recommend [installing VS Code](https://code.visualstudio.com/download) and using the
+[ms-vscode-remote.vscode-remote-extensionpack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+extension, which includes support for editing code in Docker Containers.
+
+## Development Environment
+
+### Open your `aws-template` repo in VS Code
+
+???+ warning "Warning: Windows users"
+
+    If you are using WSL on Windows then you should open VSCode from within a WSL terminal in order for VSCode so it
+    automatically installs the proper WSL interface extension.
```

Comment:
Do this by opening the correct folder in VSCode in Windows. Then hit "ctrl+shift+p" and enter the command "WSL: Reopen folder in WSL".